pages/common/jpegoptim.md

```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
```

Comment:
Eh, I don't mind either way. Personally, I'd use lowercase, but the `optipng` page uses uppercase so I'll change it that :-)